Source/Scene/Expression.js

```diff
@@ -226,6 +281,33 @@ define([
         return this._value;
     };
 
+    Node.prototype._evaluateVariableString = function(feature) {
+        var exp = this._value;
+        var result = "";
+        var i = exp.indexOf('${');
```

Comment:
I don't think that's right because you need to make sure `\$` doesn't appear, not just `\`.  You can use the debugger console to test this easily without before writing a formal test.

On the other hand, (and I could be wrong) but I believe the plan is to implement exactly what [ES6 template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings) do, so we should use that as a reference and I'm not sure how they let you escape the `$`.  @pjcozzi is my assumption here correct?